.github/workflows/ci.yml

```diff
@@ -57,3 +65,7 @@ jobs:
       run: COMMAND=./dist/kubectx_linux_amd64_v1/kubectx bats test/kubectx.bats
     - name: kubens (Go) integration tests
       run: COMMAND=./dist/kubens_linux_amd64_v1/kubens bats test/kubens.bats
+    - name: Publish Snaps to the Snap Store (edge channel)
+      run: for snap in $(ls ./dist/*.snap); do snapcraft upload --release=edge $snap; done
+      env:
+        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
```

Comment:
I think we should reuse the same name that makes it clear the secret is for snapcraft. 

I assume you'd be delivering the secret to me somehow? Would it be possible to list me as co-maintainer too? Do I need to open an account?

Reply:
What name do you want? I used the same on the documentation I found.
Related to the token value, you will need to create an account here: https://login.ubuntu.com. I will give permissions to your user to manage the application, or even transfer the application to you if you want so. After that, you need to get a token using the command like `snapcraft export-login --snaps=kubectx  --acls package_access,package_push,package_update,package_release credentials.txt` and use the content of the file as the value in the variable.

Reply:
Btw, I checked your email is already added.
![image](https://github.com/ahmetb/kubectx/assets/6025031/6ec907af-d2f4-4dd4-bc5a-4e3494700d5f)